homeassistant/components/onvif/config_flow.py

```diff
@@ -172,7 +172,10 @@ async def async_step_auth(self, user_input=None):
         return self.async_show_form(
             step_id="auth",
             data_schema=vol.Schema(
-                {vol.Required(CONF_USERNAME): str, vol.Required(CONF_PASSWORD): str}
+                {
+                    vol.Required(CONF_USERNAME): str,
+                    vol.Optional(CONF_PASSWORD, default=""): str,
```

Comment:
Why do we set an empty string as default?

Reply:
See #35904

Reply:
Please add a comment so we don't remove this in the future.